guides/elements.md

```diff
@@ -480,3 +531,25 @@ de:
       contact_form:
         color: Button Farbe
 ~~~
+
+## Notes on interacting with Elements from the Admin interface
+
+While this guide is intended for developers building a CMS site, many developer will need to train clients and customers, or interact with the admin interface themselves.
+
+While the Admin UI tries to be as intuitive as possible, there are a few functions that aren't quite as obvious.
+
+### Re-arranging Elements on a Page
+
+Collapsed elements can be re-arranged by clicking and dragging the window icon to the left of the element title.
```

Comment:
```suggestion
Elements can be re-arranged by clicking and dragging its icon to the left of the element title.
```

Reply:
Changed "the window" to "its", but I contend that elements have to be collapsed to actually be re-ordered.

Reply:
Since this is a bug, please change the text as requested.